src/meltano/cli/utils.py

```diff
@@ -428,13 +428,14 @@ def install_status_update(install_state: PluginInstallState):
         logger.info("%s %s '%s'", install_state.verb, desc, plugin.name)
 
 
-def install_plugins(
+async def install_plugins(
     project,
     plugins,
```

Comment:
This smells like it should be `*plugins` but that's probably out of scope for this PR.

Reply:
Sure, I thought it was best to leave the signature mostly the same. Curious on the philosophy of that though?

Reply:
I could be wrong but I think it's the more Pythonic approach for dealing with N positional arguments of the same type.